cuml/nearest_neighbors_demo.ipynb

```diff
@@ -144,6 +144,8 @@
    "source": [
     "## Compare Results\n",
     "\n",
+    "cuML currently uses FAISS for exact nearest neighbors search, which limits inputs to single-precision. This results in possible round-off errors when floats of different magnitude are added. As a result, it's very likely that the cuML results will not match Sciklearn's nearest neighbors exactly. You can read more in the [FAISS wiki](https://github.com/facebookresearch/faiss/wiki/FAQ#why-do-i-get-weird-results-with-brute-force-search-on-vectors-with-large-components).\n",
+    "\n",
     "### Distances"
    ]
   },
```

Comment:
It LGTM, just a small suggestion, i think it will help the users understand how we are comparing the results for indices if we add a small comment in the cell